### PR TITLE
refactor(hydro_deploy)!: rename integration crates to drop CLI references

### PR DESCRIPTION
**Pull request by shadaj**

### CODE REVIEW

File: hydro_deploy/hydroflow_plus_deploy/CHANGELOG.md [hunk 1]
```diff
@@ -81,11 +81,11 @@
...
-   clusters
-   
-   This allows `extract` to move the flow builder, which is a prerequisite
-   for having developers run the optimizer during deployment as well in
+   feat(hydroflow_plus): simplify lifetime bounds for processes and
+   clusters
+   
+   This allows `extract` to move the flow builder, which is a prerequisite
+   for having developers run the optimizer during deployment as well in
    case it changes the network topology.
```
Comment (MingweiSamuel) [inline hydro_deploy/hydroflow_plus_deploy/CHANGELOG.md:89]: can you double check that these are not CRLF line ends?

**Review by MingweiSamuel — CHANGES_REQUESTED** [review]
You will need to follow
 https://github.com/hydro-project/hydroflow/blob/main/RELEASING.md#addendum-moving-crates
 For `hydroflow_deploy_integration` and `hydroflow_plus_deploy`

File: hydro_deploy/hydroflow_plus_deploy/CHANGELOG.md (hunk 1 above)
Comment (shadaj) [reply hydro_deploy/hydroflow_plus_deploy/CHANGELOG.md:148]: Seems to be okay, this is what is generated by `cargo changelog`

**Review by MingweiSamuel — APPROVED** [review]
Looks good!

File: docs/docs/deploy/your-first-deploy.md [hunk 2]
```diff
@@ -26,20 +26,20 @@ rustup update
...
 ```
 
-We'll need to add an additional dependency for `hydroflow_cli_integration` to our `Cargo.toml`:
+We'll need to add an additional dependency for `hydroflow_deploy_integration` to our `Cargo.toml`:
 
 ```toml
 [dependencies]
 # ...
-hydroflow_cli_integration = "0.1.1"
+hydroflow_deploy_integration = "0.1.1"
```
Comment (MingweiSamuel) [inline docs/docs/deploy/your-first-deploy.md:34]: Guess we're at 0.5.2 by now

**Pull request merged**